docs/mkdocs.yml

```diff
@@ -48,15 +48,16 @@ nav:
     - flink-actions.md
     - flink-configuration.md
   - hive.md
-  - Trino: https://trino.io/docs/current/connector/iceberg.html
-  - Clickhouse: https://clickhouse.com/docs/en/engines/table-engines/integrations/iceberg
+  - Doris: https://doris.apache.org/docs/dev/lakehouse/multi-catalog/iceberg
+  - Impala: https://impala.apache.org/docs/build/html/topics/impala_iceberg.html
   - Presto: https://prestodb.io/docs/current/connector/iceberg.html
-  - Dremio: https://docs.dremio.com/data-formats/apache-iceberg/
   - Starrocks: https://docs.starrocks.io/en-us/latest/data_source/catalog/iceberg_catalog
+  - Trino: https://trino.io/docs/current/connector/iceberg.html
   - Amazon Athena: https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg.html
   - Amazon EMR: https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-iceberg-use-cluster.html
-  - Impala: https://impala.apache.org/docs/build/html/topics/impala_iceberg.html
-  - Doris: https://doris.apache.org/docs/dev/lakehouse/multi-catalog/iceberg
+  - Clickhouse: https://clickhouse.com/docs/en/engines/table-engines/integrations/iceberg
+  - Dremio: https://docs.dremio.com/data-formats/apache-iceberg/
```

Comment:
This new ordering confuses me a bit, I believe it's something to discuss as I believe it should just be alphanumeric.

@Fokko @RussellSpitzer @nastra @flyrain @ajantha-bhat, is there any reason we shouldn't just make these alphanumeric?

I believe Spark, Flink, and Trino have the most support and why they came first initially, but as this project grows, it's likely better we just follow alphanumeric everywhere. Unless we want to group engines by some metric of Iceberg support and then alphanumeric.

Let me know what you all think and if this should be a topic at the next sync.

Reply:
Alphanumeric order makes sense to me. But it may mess up with other non-engine pages like `Table`, `API`. Should we introduce another top-level index to group all engines(Spark, Flink, Snowflake, PyIceberg, etc)?